src/client/components/Bruincast/index.js

```diff
@@ -140,6 +158,16 @@ export const Bruincast = ({ course, warning, retrieveWarning, userid }) => {
         Video recordings may take up to 24 hours before they are available.{' '}
         <Link href="#">Help</Link>
       </Text>
+      <br />
+      <div>
+        <Button
+          renderIcon={ascendingSort ? IconArrowUpLine : IconArrowDownLine}
```

Comment:
@chern It's a dead heat on the poll.. But let's just stick with the arrows you have here.